api/app/Models/Pool.php

```diff
@@ -122,29 +124,19 @@ public function screeningQuestions(): HasMany
     public function getStatusAttribute()
     {
         // given database is functioning in UTC, all backend should consistently enforce the same timezone
```

Comment:
This comment is from before, but I don't think it is needed for the current state of the accessor and makes it slightly more confusing. Please delete it.

Reply:
:+1: 
[remove old comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/a8fde02611df2a8b462951178d6c4329ffc13e7b)